IBSWeb/Areas/Filpride/Views/Report/GeneratedGmReport.cshtml

```diff
@@ -0,0 +1,228 @@
+@model IEnumerable<FilprideReceivingReport>
+@using Microsoft.AspNetCore.Identity
+@inject SignInManager<IdentityUser> SignInManager
+@inject UserManager<IdentityUser> UserManager
+@{
+    ViewData["Title"] = "GM Report";
+
+    string currencyFormat = "N4";
+    var totalVolume = 0m;
+    decimal? averagePurchasePrice = 0m;
```

Comment:
Find a way to lessen the initialization like this, the decimal should be not nullable because the default value of this type was zero